app/views/shared/_header.html.slim

```diff
@@ -11,3 +11,5 @@ nav.navbar.navbar-dark.bg-primary.navbar-expand-lg.fixed-top
         - if current_user&.student?
           li.nav-item = link_to t('.profile'), new_user_session_path, class: 'nav-link'
           li.nav-item = link_to t('.profile_edit'), new_user_registration_path, class: 'nav-link'
+          - if profile
+            li.nav-item = link_to cl_image_tag(profile.avatar.medium)
```

Comment:
Rebase against master needed, `header` now in `app/views/application/_header.html.slim` and for logged in user there is an `app/views/application/_user_nav.html.slim` partial